AutomatedTesting/Gem/PythonTests/Prefab/tests/create_prefab/CreatePrefab_WithNestedEntitiesAndNestedPrefabs.py

```diff
@@ -0,0 +1,100 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def CreatePrefab_WithNestedEntitiesAndNestedPrefabs():
+    """
+    Test description:
+    - Creates linear nested entities. 
+    - Creates a prefab from the nested entities.
+
+    Test passed if the newly instanced prefab of the 3 entities have correct hierarchy and positions.
+    """
```

Comment:
This test has the exact same test description as CreatePrefab_WithNestedEntities